crates/aranya-client-capi/src/api/defs.rs

```diff
@@ -928,3 +992,197 @@ pub unsafe fn afc_recv_data(
 
     Ok(true)
 }
+
+/// Query devices on team.
+///
+/// @param client the Aranya Client [`Client`].
+/// @param team the team's ID [`TeamId`].
+/// @param devices returns a list of device IDs on the team [`DeviceId`].
+/// @param devices_len returns the length of the devices list [`DeviceId`].
+///
+/// @relates AranyaClient.
+pub fn query_devices_on_team(
+    client: &mut Client,
+    team: &TeamId,
+    devices: Option<&mut MaybeUninit<DeviceId>>,
+    devices_len: &mut usize,
+) -> Result<(), imp::Error> {
+    let client = client.deref_mut();
+    let data = client
+        .rt
+        .block_on(client.inner.queries(team.0).devices_on_team())?;
+    let data = data.__data();
+    if let Some(devices) = devices {
+        let out = aranya_capi_core::try_as_mut_slice!(devices, *devices_len);
+        for (dst, src) in out.iter_mut().zip(data) {
+            dst.write(DeviceId(*src));
+        }
+        *devices_len = data.len();
+        return Ok(());
+    }
+    *devices_len = data.len();
+    Ok(())
+}
+
+/// The formula for computing the amount of characters in a base64 string from the original number of bytes is:
+/// base64_str_len = (bytes*1375)/1000
+///
+/// A DeviceId is 64 bytes.
+///  
+/// ARANYA_DEVICE_ID_STR_LEN is the number of characters required to hold the base64 string plus the null terminator:
+/// (sizeof(AranyaDeviceId)*1375)/1000+1 = 89
+// TODO: compute size from size_of::<DeviceId>()
+pub const ARANYA_DEVICE_ID_STR_LEN: u64 = (64 * 1375) / 1000 + 1;
+
+/// Returns a human-readable message for a [`DeviceId`].
+///
+/// This method converts the DeviceId to a base64 encoded string.
+///
+/// Before calling this method, allocate a string of size ARANYA_DEVICE_ID_STR_LEN.
```

Comment:
These docs are still wrong (it doesn't return anything, it's not a message, it doesn't use base64, etc.).

How about:

```rust
/// Writes the human-readable encoding of `device` to `str`.
///
/// To always succeed, `str` must be at least `ARANYA_DEVICE_ID_STR_LEN` bytes long.
```